### ISSUE TITLE
Issue with the values of the ratings in test set

### ISSUE THREAD

**Issue by HaikelMarie** [NicolasHug/Surprise#245]
Hi,

When running the example on Surprise's GitHub (and also working on my
own dataset), I noticed that, when building a test set, the ratings in
the test set all have the same value.

Here is some code to reproduce the issue:

from surprise import SVD
from surprise import Dataset
from surprise.model_selection import cross_validate

data = Dataset.load_builtin('ml-100k')
trainset = data.build_full_trainset()
testset = trainset.build_anti_testset()

print(np.unique(np.array([r for _, _, r in testset()])))

Is there something I don't understand?

Thanks,

**Comment by NicolasHug** [comment]
As you can see in the doc of `build_anti_testset` https://surprise.readthedocs.io/en/stable/trainset.html#surprise.Trainset.build_anti_testset

the ratings are unknown, by definition.

If you want to split your dataset into differents sets where the ratings are known you'll need to use either CV iterators or `train_test_split`.

https://surprise.readthedocs.io/en/stable/getting_started.html#train-test-split-and-the-fit-method
